lib/services/project-data-service.ts

```diff
@@ -63,7 +63,10 @@ export class ProjectDataService implements IProjectDataService {
 					"description": "NativeScript Application",
 					"license": "SEE LICENSE IN <your-license-filename>",
 					"readme": "NativeScript Application",
-					"repository": "<fill-your-repository-here>"
+					"repository": "<fill-your-repository-here>",
+					"devDependencies": {
```

Comment:
why the code is here? It can be added to the templates as all devDependencies from the template will be added to the currently created project:
https://github.com/NativeScript/nativescript-cli/blob/master/lib/services/project-service.ts#L76

https://github.com/NativeScript/nativescript-cli/blob/master/lib/services/project-service.ts#L100-L102